CHANGELOG.md

```diff
@@ -5,6 +5,14 @@ All notable changes to this project will be documented in this file.
 The format is _loosely_ based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/). The project does _not_ follow
 Semantic Versioning and the changes are simply documented in reverse chronological order, grouped by calendar month.
 
+# August 2024
+
+## com.mbeddr.mpsutils
```

Comment:
This is not the name of the language.

Reply:
Please check again.
I am unsure cause what I mentioned is the solution now:

![image](https://github.com/user-attachments/assets/e9df0fae-5fe5-4786-9f51-6f1cc2d2a453)